content/admin/authentication/using-ldap.md

```diff
@@ -98,6 +98,12 @@ When this option is selected, the certificate is validated to make sure:
 
 #### Enabling LDAP Sync
 
+{% note %}
+
+**Note:** LDAP synchronized teams are limited to a maximum 1499 members.
```

Comment:
```suggestion
**Note:** Teams using LDAP Sync are limited to a maximum 1499 members.
```
To keep it consistent with the rest of the article, let's call it LDAP Sync (I used `command+f` in the article to look for the word "synchronized" and it doesn't look like it's used anywhere else). 

Then, to avoid stacked modifiers (from the [localization guide](https://github.com/github/docs/blob/main/contributing/localization-checklist.md#examples)) we can switch it around to "Teams using LDAP sync."

Reply:
Indeed, much clearer. Thanks!